src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -233,8 +233,8 @@ def applyPsetTweak(self, psetTweak, skipIfSet=False, allowFailedTweaks=False, na
             cmd += " --allow_failed_tweaks"
         self.scramRun(cmd)
 
-        if cleanupTweak is True:
-            psetTweak = PSetTweak()
+        if cleanupTweak:
```

Comment:
Kenyi, could you please create a docstring for this method and clarify what all these options are? For instance, I do not even know why we would want to reset the PSet tweak :) I'm sure this will help Stefano as well.